lock-api-objects/src/test/java/com/palantir/lock/watch/LockEventTest.java

```diff
@@ -0,0 +1,219 @@
+import com.palantir.lock.v2.LockToken;
+import com.palantir.logsafe.exceptions.SafeIllegalArgumentException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import org.immutables.value.Value;
+import org.junit.Test;
+
+public class LockEventTest {
+    private static final String BASE = "src/test/resources/lock-event-wire-format/";
+    private static final boolean REWRITE_JSON_BLOBS = false;
+
+    private static final LockDescriptor LOCK_1 = StringLockDescriptor.of("abc");
+    private static final LockDescriptor LOCK_2 = StringLockDescriptor.of("def");
+    private static final LockDescriptor LOCK_3 = StringLockDescriptor.of("ghi");
+    private static final LockDescriptor LOCK_4 = StringLockDescriptor.of("jkl");
+    private static final long SEQUENCE = 10L;
+    private static final LockToken LOCK_TOKEN = LockToken.of(new UUID(1337, 42));
+    private static final Set<LockDescriptor> LOCK_SET = ImmutableSet.of(LOCK_1, LOCK_2, LOCK_3, LOCK_4);
+    private static final OldLockEvent OLD_LOCK_EVENT = ImmutableOldLockEvent.builder()
+            .sequence(SEQUENCE)
+            .lockDescriptors(LOCK_SET)
+            .lockToken(LOCK_TOKEN)
+            .build();
+    private static final LockEvent BASELINE_LOCK_EVENT = ImmutableLockEvent.builder()
+            .sequence(SEQUENCE)
+            .lockDescriptors(LOCK_SET)
+            .lockToken(LOCK_TOKEN)
+            .build();
+    private static final byte[] BYTES_OLD = PtBytes.toBytes("old");
+    private static final byte[] BYTES_NEW = PtBytes.toBytes("new");
+    private static final byte[] BYTES_DELETED = PtBytes.toBytes("deleted");
+    private static final byte[] BYTES_CREATED = PtBytes.toBytes("created");
+    private static final LockRequestMetadata LOCK_REQUEST_METADATA = LockRequestMetadata.of(ImmutableMap.of(
+            LOCK_1,
+            ChangeMetadata.unchanged(),
+            LOCK_2,
+            ChangeMetadata.updated(BYTES_OLD, BYTES_NEW),
+            LOCK_3,
+            ChangeMetadata.deleted(BYTES_DELETED),
+            LOCK_4,
+            ChangeMetadata.created(BYTES_CREATED)));
+
+    // TimeLock (Server) serializes and AtlasDB (Client) deserializes.
+    // These are the respective mappers used internally by Conjure.
+    private static final ObjectMapper SERIALIZATION_MAPPER =
+            ObjectMappers.newServerObjectMapper().enable(SerializationFeature.INDENT_OUTPUT);
+    private static final ObjectMapper DESERIALIZATION_MAPPER =
+            ObjectMappers.newClientObjectMapper().enable(SerializationFeature.INDENT_OUTPUT);
+    // This mapper is used to ensure that two JSONs are equal excluding indentation
+    private static final ObjectMapper VERIFYING_MAPPER = new ObjectMapper();
+
+    @Test
+    public void oldLockEventIsBaseline() {
```

Comment:
tiny nit: I'd maybe call this something outside of "baseline", as the baseline will change after this PR.